example/python/permissions/commons.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import iroha
+from time import time
+
+
+def now():
+    return int(time() * 1000)
+
+
+def all_permissions():
+    return iroha.RolePermissionSet([
```

Comment:
```python
r = iroha.RolePermissionSet()
for i in range(r.size()):
  r.set(i)
return i
```

Reply:
Discussed offline. Since the listing of commons.py will also be included in the documentation page about permissions, it is better to preserve fully-qualified constants' names for better presentation for documentation users.